ruby-challenges.rb

```diff
@@ -1,30 +1,63 @@
 # ASSESSMENT 5: Ruby Coding Practical Questions
 # MINASWAN
 
-# --------------------1) Create a method that takes in an array of words and a single letter and returns an array of all the words containing that particular letter. Use the test variables provided.
 
-beverages_array = ['coffee', 'tea', 'juice', 'water', 'soda water']
 
-letter_o = 'o'
+# 1) Create a method that takes in an array of words and a single letter and returns an array of all the words containing that particular letter. Use the test variables provided.
+
+#beverages_array = ['coffee', 'tea', 'juice', 'water', 'soda water']
+
+#letter_o = 'o'
 # Expected output: ['coffee', 'soda water']
-letter_t = 't'
+#letter_t = 't'
 # Expected output: ['tea', 'water', 'soda water']
+#beverages_array = ['coffee', 'tea', 'juice', 'water', 'soda water']
+#Pseudo Code
+# Use select method on beverages_array so I can filter the elements in the array based on the conditions I want.
+# Save the results of the filtered out elements into a new array called results that'll print out the well results.
+
+beverages_array = ['coffee', 'tea', 'juice', 'water', 'soda water']
+result = beverages_array.select{ |beverage| beverage.include?('t')}
+print result
+    
+#This code works!. Just change the letter that will be used to filter out the words that don't have it.
```

Comment:
There is no method being created here. Remember we use the keyword `def` to declare a method, and also instead of using the given variables, we use parameters as placeholders for those variables.